packages/fast-development-site-react/src/components/site/configuration-panel.tsx

```diff
@@ -173,6 +173,15 @@ class ConfigurationPanel extends React.Component<
         );
     }
 
+    private isChildComponent(): boolean {
+        const dataLocationSegments: string[] = this.props.dataLocation.split(".");
+
+        return (
+            this.props.dataLocation === "" ||
```

Comment:
could also use `this.props.dataLocation.endsWith("props")` and avoid splitting the string.